its/ruling/src/test/resources/expected/python-S5795.json

```diff
@@ -4,11 +4,6 @@
 217,
 ],
 'project:numpy-1.16.4/numpy/distutils/misc_util.py':[
-340,
-341,
-342,
-343,
-344,
```

Comment:
Fixed FP! in Python 2 it `curses.tigetstr()` returns a `bytes` which is an alias of `str`.

Reply:
Just to make sure, is this because the rule only raise on the `str` constructor but not on the `str` type?
As a side note, I think we should introduce an exception for this rule when one of the operand is `None`, as the intend in that case is probably not to have an accurate comparison (and the other rule about redundant comparison to None would apply anyway) WDYT?